tests/python/contrib/test_hexagon/pytest_util.py

```diff
@@ -155,5 +166,15 @@ def create_populated_numpy_ndarray(
     elif type(itp) == TensorContentRandom:
         return np.random.random(input_shape).astype(dtype)
 
+    elif type(itp) == TensorContentSequentialCOrder:
+        a = np.empty(tuple(input_shape), dtype)
+
+        with np.nditer(a, op_flags=["writeonly"], order="C") as it:
```

Comment:
wondering if this should get a unittest

Reply:
> is should get a unittest

I was on the fence about that.  I did test it manually, but I thought unit testing a little debug-assist like this might be overkill.